WebService/v1/Controllers/SimulationsController.cs

```diff
@@ -55,7 +58,10 @@ public async Task<SimulationListApiModel> GetAsync()
         [HttpGet("{id}")]
         public async Task<SimulationApiModel> GetAsync(string id)
         {
-            return SimulationApiModel.FromServiceModel(await this.simulationsService.GetAsync(id));
+            var simulation = await this.simulationsService.GetAsync(id);
+            var stats = await this.simulationStatistics.GetSimulationStatisticsAsync(id);
```

Comment:
can we merge these 2 calls, something like:
```
var simulation = await this.simulationService.GetWithStatistics(id)
```

Reply:
done